pkg/users/account-details.js

```diff
@@ -164,30 +188,14 @@ export function AccountDetails({ accounts, groups, current_user, user, shells })
                 .catch(show_unexpected_error);
     }
 
-    if (!accounts.length) {
+    if (isLoading) {
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21865-f1a18461-20250828-203100-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)